pkg/machinery/scaffold.go

```diff
@@ -370,10 +370,7 @@ func filterExistingValues(content string, codeFragmentsMap CodeFragmentsMap) err
 			}
 		}
 	}
-	if err := scanner.Err(); err != nil {
```

Comment:
FWIW I do not like this linter because it leads to more diff lines when changing downstream execution within a function body in later changes. However this seems to be a source of several issues so I do not want to block this from merging.